src/handlers/v1/markets/[chainId]/[marketId]/zap-borrow/index.ts

```diff
@@ -16,9 +16,9 @@ import { validateMarket } from 'src/validations';
 
 type GetZapBorrowQuotationRouteParams = EventPathParameters<{ chainId: string; marketId: string }> &
   EventBody<{
-    account?: string;
-    amount?: string;
-    targetToken?: common.TokenObject;
+    account: string;
+    baseAmount?: string;
```

Comment:
```suggestion
    srcAmount?: string;
```
I just noticed this, although not in the Asana comments. I'd like to discuss changing it to `srcAmount` since 'borrow' is only to the base token without ambiguity like `leverage` or `deleverage`. It helps all zaps having the similar parameters.

Reply:
Not bad. Remember to update previous discussions also.